modules/lang-painless/build.gradle

```diff
@@ -51,18 +51,6 @@ tasks.named("test").configure {
 
 tasks.named("yamlRestCompatTest").configure {
     systemProperty 'tests.rest.blacklist', [
-            'painless/70_execute_painless_scripts/Execute with date field context (single-value)',
```

Comment:
these were not type related. Those were added temporarily before backporting the changes to 7.x
https://github.com/elastic/elasticsearch/pull/72135/files#r619034432